tests/src/common/Wsk.scala

```diff
@@ -67,21 +67,25 @@ import java.time.Instant
  * It also sets the apihost and apiversion explicitly to avoid ambiguity with
  * a local property file if it exists.
  */
+
+
 case class WskProps(
     authKey: String = WhiskProperties.readAuthKey(WhiskProperties.getAuthFileForTesting),
     namespace: String = "_",
     apiversion: String = "v1",
     apihost: String = WhiskProperties.getEdgeHost) {
-    def overrides = Seq("--apihost", apihost, "--apiversion", apiversion)
+    def overrides = Seq("-i", "--apihost", apihost, "--apiversion", apiversion)
 }
 
-class Wsk extends RunWskCmd {
+class Wsk(val usePythonCLI: Boolean = true) extends RunWskCmd {
     implicit val action = new WskAction()
     implicit val trigger = new WskTrigger()
     implicit val rule = new WskRule()
     implicit val activation = new WskActivation()
     implicit val pkg = new WskPackage()
     implicit val namespace = new WskNamespace()
+
+    Wsk.setUsePythonCLI(usePythonCLI)
```

Comment:
this is going to be a problem when you flip test suites from python to go since the variable is "global" in that it's part of the singleton. i think instead the methods of the singleton affected by the switch have to accept an additional argument (use python cli) so that the variable is scoped to the method instance called not the static object.